source/partials/drupal-8-convert-to-composer.md

```diff
@@ -0,0 +1,255 @@
+
+### Custom Code
+
+Manually copy custom code from the existing site repository to the Composer-managed directory.
+
+#### Modules and Themes
+
+Modules:
+
+```bash{promptUser:user}
+mkdir -p ../$SITE-composer/web/modules/custom # create the directory if it doesn't already exist
+cp -r modules/custom/awesome_module ../$SITE-composer/web/modules/custom
+```
+
+Themes:
+
+```bash{promptUser:user}
+mkdir -p ../$SITE-composer/web/themes/custom # create the directory if it doesn't already exist
+cp -r themes/custom/great_theme ../$SITE-composer/web/themes/custom
+```
+
+Follow suit with any other custom code you need to carry over.
+
+#### Settings.php
+
+Your existing site may have customizations to `settings.php` or other configuration files. Review these carefully and extract relevant changes from these files to copy over. Always review any file paths referenced in the code, as these paths may change in the transition to Composer.
+
+We don't recommend that you completely overwrite the `settings.php` file with the old one, as it contains customizations for moving the configuration directory you don't want to overwrite, as well as platform-specific customizations.
+
+```bash{promptUser:user}
+git status # Ensure working tree is clean
+git show master:sites/default/settings.php > web/sites/default/original-settings.php
+diff -Nup --ignore-all-space web/sites/default/settings.php web/sites/default/original-settings.php
+# edit web/sites/default/settings.php and commit as needed
+rm web/sites/default/original-settings.php
+```
+
+The resulting `settings.php` should have no `$databases` array.
+
+#### Configuration
+
+If you are using an exported config, you will need to move the configuration files to a new location. The preferred (and assumed) location of the configuration directories when using a nested docroot and Composer is at the root of the repository next to the web directory:
+
+```none
+ site-composer
+|-web
+|-config    <--Here!
+|-vendor
+|-composer.json
+|-etc...
+```
+
+Locate the configuration files in the existing site and move them here. If they are stored in the files directory on your existing site, retrieve them via [SFTP](/sftp), as the Git clone would not contain them. The example project is configured to use this location.
+
+## Deploy
+
+You've now committed the code to the local branch. Deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize the site's logs via `terminus drush $SITE.composerify -- wd-show` to inspect the watchdog logs, or follow the directions in our documentation on [log collection](/logs).
+
+### Deploy to a Multidev
+
+Push the changes to a Multidev called `composerify` to safely test the site without affecting the Dev environment:
+
+```bash{promptUser:user}
+git add .
+git commit -m "convert to composer"
+git push origin composerify && terminus env:create $SITE.dev composerify
```

Comment:
```suggestion
git push -u origin composerify && terminus env:create $SITE.dev composerify
```

You probably want the `-u` flag here so that the `composerify` will track the remote `composerify` branch.

Also, I think we called our remote `ic` instead of `origin` earlier.